modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -93,12 +101,21 @@ export default {
         data: {},
         hasErrors: false
       },
+      docUtilityFields: {
```

Comment:
`doc` is split into 2 parts, each one will be validated separately, avoiding having 2 `AposSchema` with the same fields but different data (see `v-model` of each `AposSchema`).